app/actions/uploadInventory.ts

```diff
@@ -538,12 +544,13 @@ const checkSampleTreesAndUpload = async (inventory: any) => {
         } else if (response && response.coordinates[0].status === 'complete') {
           sampleTree.status = SYNCED;
           sampleTree.locationId = response.id;
+          sampleTree.hid = response.hid;
```

Comment:
- [ ] I am not getting an HID codes displayed on iOS simulator while testing this PR - not for sample trees and not for the registration itself:

![Simulator Screen Shot - iPhone 13 - 2022-01-15 at 17 10 58](https://user-images.githubusercontent.com/1532418/149628991-3fb707ae-3917-43af-a973-2b6c6797dfe5.png)

Reply:
At least after reloading the data from the server all HID's are displayed.